### PR TITLE
updated background color for picker network

### PR DESCRIPTION
**Pull request by NidhiKJha**
## Explanation

This PR is to update the background color of Picker Network from `BACKGROUND_DEFAULT` to `BACKGROUND_ALTERNATIVE` as per the Figma. 

### Before
Screenshot 2022-11-11 at 6 13 06 PM

### After

Screenshot 2022-11-11 at 6 12 32 PM

### CODE REVIEW

**Review by georgewrmarshall — COMMENTED** [review]
Looking good! Made a couple suggestions

File: ui/components/component-library/picker-network/picker-network.scss [hunk 1]
```diff
@@ -3,4 +3,8 @@
 
   max-width: fit-content;
   height: var(--picker-network-height);
+
+  &:active {
+    background-color: #d6d9dc; //TODO: Add this color in design system for pressed state
```
[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/components/component-library/picker-network/picker-network.scss:8]: Design tokens have been updated now so you can use 

```suggestion
    background-color: var(--color-background-default-hover);
```

**Review by georgewrmarshall — COMMENTED** [review]
Looks good just need to update snapshots

**Review by georgewrmarshall — DISMISSED** [review]

**Review by georgewrmarshall — APPROVED** [review]
lgtm

**Review by garrettbear — APPROVED** [review]

**Pull request merged**